sylt-parser/src/statement.rs

```diff
@@ -457,30 +456,37 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
                         }
                         let mut tuple = Vec::new();
                         if matches!(ctx.token(), T::LeftParen) {
-                            ctx = ctx.skip(1);
+                            let (ctx_, skip_newlines) = ctx.push_skip_newlines(true);
+                            ctx = ctx_.skip(1);
                             loop {
                                 match ctx.token() {
                                     T::RightParen | T::EOF => break,
                                     _ => {
                                         let (ctx_, ty) = parse_type(ctx)?;
                                         tuple.push(ty);
                                         ctx = ctx_;
-                                        if !matches!(ctx.token(), T::Comma | T::RightParen) {
+                                        if !matches!(
+                                            ctx.token(),
+                                            T::Comma | T::Newline | T::RightParen
+                                        ) {
                                             raise_syntax_error!(ctx, "Expected a deliminator ','");
                                         }
+                                        ctx = ctx.skip_if(T::Newline);
```

Comment:
aren't we already skipping newlines here?

Reply:
alright so it says outdated but that's since i left the comment on a specific commit. the relevant code is on the latest commit as well

Reply:
I'm gonna merge in the rest of the enum PRs into this - so it's moved in all at once.

Or is it smarter to merge these one at a time?

Reply:
what other PRs? but yes, separate are nice if they are sufficiently standalone imo.

Reply:
I skipped making other PRs... Cause I... Erm...